grammar/grammar.y

```diff
@@ -1,15 +1,21 @@
-Root <- skip container_doc_comment? ContainerMembers eof
+Root <- skip container_doc_comment? StructMembers eof
 
 # *** Top level ***
-ContainerMembers
-    <- (
-         TestDecl ContainerMembers
-         / TopLevelComptime ContainerMembers
-         / doc_comment? KEYWORD_pub? TopLevelDecl ContainerMembers
-         / ContainerField COMMA ContainerMembers
-         / ContainerField
-         /
-     )
+StructMembers <- ContainerDeclarations StructFieldList ContainerDeclarations
```

Comment:
I think this is changing the grammar as currently declarations after fields are not allow if the last field lacks a trailing comma. For example, this won't parse:
```zig
const Foo = struct {
    a: u42,
    b: u42
    const c = 42;
};

Reply:
Ah, great catch. I've fixed this for the full pull request, but if it's decided that only the first commit is wanted, then I'll have to patch that up later.